contrib/docker/docker-compose.yml

```diff
@@ -42,6 +42,8 @@ services:
       dockerfile: contrib/docker/Dockerfile
     restart: unless-stopped
     environment:
+      # Add Mapxbox access token to correctly display deck.gl visualizations
+      #MAPBOX_API_KEY:
```

Comment:
Having private credentials written to a file that's tracked in source control makes it likely that those credentials will be leaked unintentionally later. What do you think about leveraging `docker-compose`'s support for `.env` files (described here https://docs.docker.com/compose/environment-variables/) and providing a `.env.sample` file that can be copied to a git-ignored `.env` location by the user instead?

Reply:
I wasn't aware of the support for environment variables in `docker-compose up`. Checking the docker folder, there is already a `.env` file there, and it is being tracked. That said, what do you think about keeping the `MAPBOX_API_KEY` as part of the `docker-compose` file, but modifying it to read the value from an environment variable? The result will look like this: 
`MAPBOX_API_KEY: ${MAPBOX_API_KEY}`.

In the README file, we add a note to inform the user about this option and how to set the variable:
```bash
$ export MAPBOX_API_KEY=<your_mapbox_api_key>
```
If you think it is better to have the `.env` file, another option could be specifying a "custom" environment file using the `env_file` option and add this file to the `.gitignore`.

Reply:
Adding it as an environment variable would work for me! My main concern is having it live outside of a source-controlled file. Thanks for thinking it through!